sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -179,7 +179,8 @@ case class DescribeDatabaseCommand(
 case class DropTableCommand(
     tableName: TableIdentifier,
     ifExists: Boolean,
-    isView: Boolean) extends RunnableCommand {
+    isView: Boolean,
+    purge: Boolean) extends RunnableCommand {
 
```

Comment:
minor but it would be nice to have an `assert(!isView || !purge, "Cannot purge a view")`.  presumably guaranteed by the parser anyway, but it would be nice to have it codified here.